### PR TITLE
ftests: Add cgexec test for empty cgroup

### PR DESCRIPTION
**Pull request by kamalesh-babulal**
Add a cgexec test for attaching a task to an empty cgroup without any
controller.
```
-----------------------------------------------------------------
Test Results:
        Run Date:                          Apr 21 04:54:42
        Passed:                                  1 test(s)
        Skipped:                                 0 test(s)
        Failed:                                  0 test(s)
-----------------------------------------------------------------
Timing Results:
        Test                                   Time (sec)
        -------------------------------------------------
        setup                                        0.00
        046-cgexec-empty_controller.py               2.09
        teardown                                     0.00
        -------------------------------------------------
        Total Run Time                               2.09
```
Signed-off-by: Kamalesh Babulal <kamalesh.babulal@oracle.com>

### CODE REVIEW

File: ftests/046-cgexec-empty_controller.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+
+def test(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    config.process.create_process_in_cgroup(config, '', CGNAME)
+    output = Cgroup.get(
+                         config, controller=None, cgname=CGNAME,
+                         setting='cgroup.procs', print_headers=True,
```
Comment (drakenclimber) [inline ftests/046-cgexec-empty_controller.py:42]: `print_header=True` will cause there to always be some text in `output`.  This means that the `len(output)` check will never fail.

Comment (drakenclimber) [reply ftests/046-cgexec-empty_controller.py:42]: I changed it to False.  Let me know if you have any concerns with this change.

**Comment by drakenclimber** [comment]
Pushed to main in commit 0d8f81ff854e6673029cf505cbc903248442b5c3

File: ftests/046-cgexec-empty_controller.py (hunk 1 above)
Comment (kamalesh-babulal) [reply ftests/046-cgexec-empty_controller.py:42]: @drakenclimber Thanks for the review and fixing the `print_headers`.
